tests/intern.js

```diff
@@ -1,18 +1,23 @@
 /* jshint dojo:true */
-define({
-	proxyPort: 9000,
-	proxyUrl: 'http://localhost:9000/',
-	maxConcurrency: 3,
-	loaderOptions: {
-		packages: [
-			{ name: 'digdug', location: '.' }
-		]
-	},
-	reporters: [ 'Console' ],
-	suites: [
-		'dojo/has!host-node?digdug/tests/unit',
-		'dojo/has!host-node?digdug/tests/integration/all'
-	],
-	functionalSuites: [],
-	excludeInstrumentation: /^(?:tests|node_modules)\//
+define([
+	'intern/dojo/has'
+], function (has) {
+	if (process.env.TRAVIS_EVENT_TYPE === 'cron' || process.env.INTEGRATION === 'true') {
```

Comment:
For simplicity, I'd prefer to just have the presence of the service auth parameters determine whether the integration tests should run. Integration tests can look for their specific service parameters, and just skip if the parameters aren't present. That gives the user more control over which integration tests are run, and avoids the need for more environment variables and the custom `has` flag.